src/uk/gov/hmcts/contino/DockerImage.groovy

```diff
@@ -28,14 +28,16 @@ class DockerImage {
   Acr acr
   def commit
   def registryHost
+  def lastcommittime
 
 
-  DockerImage(product, component, acr, tag, commit) {
+  DockerImage(product, component, acr, tag, commit, lastCommitTime) {
```

Comment:
don't forget to update the other references

Reply:
yup - sorted now, merged your recommendation rather than update all at once :D